### DISCUSSION TITLE
使用cookie登录程序

### DISCUSSION THREAD

**Discussion by WittF** [mikumifa/biliTickerBuy#119] [聊天专区]
用[Tsuk1ko/bilibili-qr-login](https://github.com/Tsuk1ko/bilibili-qr-login)项目的[哔哩哔哩cookie获取工具](https://bql.lolicon.app/)扫描登录获取到原文cookie
复制放到我花了10分钟用gpt搓的这个[cookie to json转换工具](https://c2j.wittf.site/)里面转换格式, 
js源码 [c2j转换工具.zip](https://cr.wittf.site/s/QlUE) （前端简陋不要在意
转换后的json可以直接拿到登录管理页面导入, 或放进`/_internal/cookies.json`

> [浏览器驱动安装出问题的解决方式](https://github.com/mikumifa/biliTickerBuy/wiki/%E6%B5%8F%E8%A7%88%E5%99%A8%E9%A9%B1%E5%8A%A8%E5%AE%89%E8%A3%85%E5%87%BA%E9%97%AE%E9%A2%98%E7%9A%84%E8%A7%A3%E5%86%B3%E6%96%B9%E5%BC%8F)

**Comment by WittF** [comment]
实测可行, 欢迎提问

**Reply by WittF** [reply]
![AAE97A6A4278AA39284F5DDE326CFE6D](https://github.com/mikumifa/biliTickerBuy/assets/108567138/33a70dca-c731-4e8b-8e7e-66b9b1c85403)
此处导入

**Comment by mikumifa** [comment]
不错的，驱动下载出问题的解决方法

**Comment by mikumifa** [comment]
v2.8.3之后对cookie结构进行了，修改，想问一下能否更改一下c2j的结构 参考 #140

**Comment by mikumifa** [comment]
已经改了，那没事了

**Reply by WittF** [reply]
> 已经改了，那没事了

哈哈😎
